NEWS

```diff
@@ -1,3 +1,8 @@
+Magpie 2.2
+----------
+With sbatch & srun, add experimental support for distributed
+ Tensorflow and Tensorflow with Horovod on top of MPI support.
+
```

Comment:
Instead, keep my original line, and add

`Add new submission script type "sbatchmpirun"`

Then

`With sbatch & mpirun, add experimental support for Tensorflow with Horovod`